tutorials/io/plot_30_reading_fnirs_data.py

```diff
@@ -61,16 +62,158 @@
 will also read the ``digaux`` data and create annotations for any triggers.
 
 
-Storing of optode locations
-===========================
-
-NIRs devices consist of light sources and light detectors.
-A channel is formed by source-detector pairs.
-MNE stores the location of the channels, sources, and detectors.
-
-
-.. warning:: Information about device light wavelength is stored in
-             channel names. Manual modification of channel names is not
-             recommended.
-
+Loading legacy data in csv or tsv format
+========================================
+
+.. warning:: This method is not supported. You should convert your data
+             to the `SNIRF <https://github.com/fNIRS/snirf>`_
+             format using the tools provided by the Society
+             for functional Near-Infrared Spectroscopy, and then load it
+             using :func:`mne.io.read_raw_snirf`.
+
+             This method will only work for data that has already been
+             converted to oxyhaemoglobin and deoxyhaemoglobin. It will not work
+             with raw intensity data or optical density data.
+
```

Comment:
Related to the previous comment we could then delete this part if I'm not wrong.
```suggestion
```

Reply:
Typo [L111](https://github.com/mne-tools/mne-python/pull/9141/files#diff-6f0870e641c47408196eed98425a9d97656eacfeeceba1807f3a515e974b7689R111) also I think.